package.json

```diff
@@ -131,6 +131,9 @@
   "scripts": {
     "lint": "eslint \"**/*.ts\" --fix",
     "lint:ci": "eslint \"**/*.ts\" --max-warnings 0",
+    "experiment": "echo ${first} && cd ../",
+    "demo": "echo \"Hello $npm_config_first $npm_config_last\"",
+    "pack-hdf-converters": "original_dir=$(pwd) && cd ${npm_config_heimdall:-../heimdall2} && cd libs/hdf-converters/ && git switch ${branch:-master} && git fetch && yarn install && yarn pack && cd $original_dir && npm i && npm i ${npm_config_heimdall:-../heimdall2}/libs/hdf-converters/mitre-hdf-converters-v*.tgz && yarn prepack",
```

Comment:
Do you mean that we should put quotes around the ${} ?

Reply:
yeah, so wherever you use any variable (including original_dir), you should probably have it be like `"$original_dir"` but since it's in a string defined with double quotes (cause of json rules), then it would actually look like `\"$original_dir\"` when you're actually using it there.  as always test to figure out how the behavior works.  try adding a space in the path you have your saf cli and/or heimdall repos in (ex /Users/username/path/to/my location/saf)

as further reading, you should probably look into how bash handles variables and what using quotes around them does

Reply:
yeah, just put quotes around most of the vars and it seems to work